src/DeploymentProvider/AzureSpringCloudDeploymentProvider.ts

```diff
@@ -0,0 +1,202 @@
+import * as core from '@actions/core';
+import { v4 as uuidv4 } from 'uuid';
+import { Package, PackageType } from 'azure-actions-utility/packageUtility';
+import { Actions, ActionParameters, ActionParametersUtility } from '../operations/actionParameters';
+import { AppPlatformManagementClient, AppPlatformManagementModels as Models } from '@azure/arm-appplatform'
+import { getDefaultAzureCredential } from '@azure/identity'
+import { DeploymentHelper as dh } from "./DeploymentHelper";
+import * as tar from 'tar';
+
+export class AzureSpringCloudDeploymentProvider {
+
+    defaultInactiveDeploymentName = 'staging';
+
+    params: ActionParameters;
+    client: AppPlatformManagementClient;
+    logDetail: string;
+
+    constructor() {
+        this.params = ActionParametersUtility.getParameters();
+    }
+
+    public async PreDeploymentStep() {
```

Comment:
Please follow the TypeScript style guide https://google.github.io/styleguide/tsguide.html#identifiers to update the cases for identifiers.

type names should use UpperCamelCase
variables / methods / properties should use lowerCamelCase